packages/backend/src/models/CatalogModel/utils/index.ts

```diff
@@ -29,6 +30,10 @@ export const convertExploresToCatalog = (
             description: field.description || null,
             type: CatalogType.Field,
             field_type: field.fieldType,
+            required_attributes:
+                field.requiredAttributes ??
+                baseTable.requiredAttributes ??
+                null,
```

Comment:
This has the same order as we previously had in `packages/backend/src/models/CatalogModel/utils/parser.ts` line 27, field attributes come before base table